src/specfuzz_init.c

```diff
@@ -48,26 +48,48 @@ void specfuzz_handler(int signo, siginfo_t *siginfo, void *ucontext) {
 #if ENABLE_SANITY_CHECKS == 1
     if (inside_handler != 0) {
         fprintf(stderr, "\n[SF] Error: Fault inside the signal handler\n");
+#if SEED_NON_SPECULATIVE_ERRORS == 1
```

Comment:
If this case is reached, then it's definitely a bug in SpecFuzz. It **must** crash at this point with an error. There's no reason to continue fuzzing when a simulation is buggy.

Reply:
So you stop fuzzing and the latest file in corpus can be used for debugging